tests/component/system/test_physical_channel.py

```diff
@@ -37,3 +42,53 @@ def test___physical_channels_with_different_names___hash___not_equal(
 
     assert hash(phys_chan1) != hash(phys_chan2)
     assert hash(phys_chan1) != hash(phys_chan3)
+
+
+def test__physical_channel__write_to_teds_from_array_with_invalid_stream__throws_data_error(
```

Comment:
Move "invalid stream" to the given clause.

Correct terminology: "bitstream" not "stream")

This also applies to the other test cases.

`test___invalid_bitstream___write_to_teds_from_array___throws_data_error`
`test___invalid_file_path___write_to_teds_from_file___throws_data_error`